content/d3/concepts/selection/terms/csv/csv.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: '.csv()'
+Description: 'Used to load a .csv file or data and returns the data as an array of objects.'
+Subjects:
+  - 'Data Visualization'
+  - 'Data Science'
+Tags:
+  - 'Methods'
+  - 'D3'
+  - 'Data'
+CatalogContent:
+  - 'learn-d3'
+  - 'paths/data-science'
+---
+
+The **`.csv()`** method in D3 is used to load a .csv file or data and returns the data as an array of objects. A .csv file contains data in the form of `comma-separated-values`, which can be parsed by .csv() to be accessible for D3.
+
+## Syntax
+
+```pseudo
+d3.csv(url[[, accessor], callback])
+```
+
+- `url`: contains the csv data.
+- `accessor`: an optional conversion function to change the representation.
+- `callback`: an optional callback function that executes once the data is loaded.
+
+
+## Example
+
+Consider the data in workers.csv:
+
+```
+Name,   Sex,    Age,    Height (in),    Weight (lbs)
+Alex,   M,      41,     74,             170
+Elly,   F,      30,     66,             124
+Fran,   F,      33,     66,             115
+Hank,   M,      30,     71,             158
+```
```

Comment:
Change the example as the output of this CSV data has some plagiarism. Change it and add 1-2 lines of description above the code.

Reply:
Okay, do I commit and push the changes and will these appear in this pull request automatically? I shouldn't make a new pull request right? This is my first time contributing, would appreciate the help.